core/src/com/geekbrains/td/Antiking.java

```diff
@@ -0,0 +1,179 @@
+package com.geekbrains.td;
+
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.math.*;
+
+public class Antiking extends Monster {
+    private float animationTimer, timePerFrame;
```

Comment:
According to Java conventions, inline multiple declarations are not used. Each field should have it's own line. Otherwise, it looks like a logical pair or something, which shouldn't be done this way.